build.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
```

Comment:
Added comment. Short version: we want bash to report failure if anything goes wrong.